Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean

```diff
@@ -900,7 +900,7 @@ end SmoothManifoldWithCorners
 theorem LocalHomeomorph.singleton_smoothManifoldWithCorners {𝕜 : Type*} [NontriviallyNormedField 𝕜]
     {E : Type*} [NormedAddCommGroup E] [NormedSpace 𝕜 E] {H : Type*} [TopologicalSpace H]
     (I : ModelWithCorners 𝕜 E H) {M : Type*} [TopologicalSpace M] (e : LocalHomeomorph M H)
-    (h : e.source = Set.univ) :
```

Comment:
I'm a little wary of this change, it makes it harder to tell if the statement is accidentally about `Finset.univ`. Changes within the *proofs* are harmless though.

Reply:
I'd argue that this point doesn't really apply *in this case*:
- `Finset`s are never mentioned in this file
- this entire file is about manifolds: M being a `Finset` would be the utterly boring case. (Besides, I presume anybody working with local homeomorphisms would know that their source is a set (an open subset of M). *That* could only be finite if M were finite, which is altogether excedingly rare.)

That said, I've reverted this change for now, as I don't care strongly about this.